test_util/test_upgrade_vpc.py

```diff
@@ -42,62 +42,33 @@
 import os
 import pprint
 import sys
+import traceback
 import uuid
-from contextlib import contextmanager
+
+from typing import Callable, List
```

Comment:
When I try to move it up with the above group flake8 gives me the following error:
```
./test_util/test_upgrade_vpc.py:47:1: I100 Import statements are in the wrong order.import uuid should be before from typing
```